lib/reply.js

```diff
@@ -553,6 +553,7 @@ function handleError (reply, error, cb) {
       })
     : serializerFn(Object.create(error, {
       error: { value: statusCodes[statusCode + ''] },
+      code: { value: error.code },
```

Comment:
The way object.create works.. it will create a new object inheriting from the parent. It will have the property.